### ISSUE TITLE
Windows Desktop users + Ledger = error ".tmp.node is not a valid Win32 application"

### ISSUE THREAD

**Issue by tayvano** [MyCryptoHQ/MyCrypto#2064]
### Description of the Issue

Windows users are unable to interact with Ledger on Desktop Application as they get an error: `".tmp.node is not a valid Win32 application"`

Multiple reports

### Steps to Reproduce

- Download & Install Desktop app on your Windows machine

- Open app

- Click connect to ledger (doesn't matter if you have a Ledger device connected or not—error is the same

### Description of Your Machine

- Windows 10, 64 bit 

- Windows 7, 64 bit

#### Report 1: 

- https://old.reddit.com/r/MyCrypto/comments/8ydif7/mycrypto_v12_is_out_and_a_stable_desktop_app_with/e2a8lfv/

```
Installed it but I get some weird " .tmp.node is not a valid Win32 application\\? " error when trying to login with my Nano S.
```

#### Report 2: Jordan

```
\\?\C:\Users\Jordan\AppData\Local\Temp\1b247efa-2c89-4998-98bc-8061998f6361.tmp.node is not a valid Win32 application\\?\C:\Users\Jordan\AppData\Local\Temp\1b247efa-2c89-4998-98bc-8061998f6361.tmp.node
```

![ss 2018-07-12 at 05 43 04](https://user-images.githubusercontent.com/7924827/42678232-22115bbe-8633-11e8-90b7-a893a9cbad7f.png)

![ss 2018-07-12 at 05 45 40](https://user-images.githubusercontent.com/7924827/42678254-32c76d4a-8633-11e8-8f91-f59e5fdd4c52.png)

#### Report 3: via Discord

```
getting a javascript error when trying to connect to ledger nano
and __.tmp.node is not a valid win 32 application
via desktop app
```

#### Report 4: https://www.reddit.com/r/ethereum/comments/8yf6ht/mycrypto_120_is_live_desktop_app_wledger_trezor/e2b0n3l/

```
%1 is not a valid Win32 application. \\?\C:\Windows\Temp\29fbe1d7-2811-419d-ab2d-740ba9f5196d.tmp.node
```

**Comment by tayvano** [comment]
### Research and stuff we know after debugging

- Same issue is happening in latest version of Mist
    - https://github.com/ethereum/mist/issues/3988
    - https://gitter.im/ethereum/mist?at=5b3ec49ef1664406611d82f3

- Issue is **not** happening in Ledger Live (also Electron / React) 
    - https://github.com/LedgerHQ/ledger-live-desktop

**Via Connor:**

I've isolated the issue to https://github.com/MyCryptoHQ/MyCrypto/commit/6fc6d9e0745c13c7275992cb913cf12924fff0d4.

The PR before, https://github.com/MyCryptoHQ/MyCrypto/commit/3f57520bcac5a32d2d6ede979f845b355ffef475, works fine.

There are no differences between `package.json` and `yarn.lock`.

If I check out 6fc6d9e0745c13c7275992cb913cf12924fff0d4 and manually undo the code changes, the error persists.

I don't really know what to check next, I have pretty much no experience with electron and no windows machine to try things out on. 

The temporary fix, I suppose, would be to do a reverse-commit of just 6fc6d9e0745c13c7275992cb913cf12924fff0d4 if deemed necessary, otherwise I can work with another dev tomorrow to try to rebuild until we determine exactly what it is causing the issue.

**What Google Says**

- More broadly, the issue is when you have some library or dependency that is 32bit on your 64 bit machine or vice versa. 

- Error has occurred for years and years

-

**Comment by tayvano** [comment]
Fixed in v 1.2.1 

PR w/ fix: https://github.com/MyCryptoHQ/MyCrypto/pull/2067
